extensions/3DTILES_metadata/schema/3DTILES_metadata.classes.class.property.schema.json

```diff
@@ -0,0 +1,110 @@
+    },
+    "blobByteLength": {
+      "type": "integer",
+      "minimum": 1,
+      "description": "This property can only be used with a `BLOB` property or an `ARRAY` of `BLOB`. When present, all blob values for this property must be exactly this length in bytes. If the property is an `ARRAY`, then each component must have this byte length. Furthermore, no `offsetBufferView` is needed for specifying the start byte of each blob. When blob In cases where the blob is Base64 encoded in JSON (e.g. as a `defaultValue`), `blobByteLength` describes the **decoded** byte length, not the Base64 string."
+    },
+    "normalized": {
+      "type": "boolean",
+      "description": "Specifies whether integer values are normalized. This applies both when `type` is an integer type, or when `type` is `ARRAY` with a `componentType` that is an integer. For unsigned integer types, values are normalized between [0, 1]. For signed integer types, values are normalized between [-1, 1]. For all other types, this property is ignored.",
+      "default": false
+    },
+    "max": {
+      "type": "array",
+      "description": "Maximum allowed values for property values. Only applicable for fixed-size numeric element types (integer, floats and `ARRAY`s thereof). Both `min` and `max` arrays have the same length. When `type` is `ARRAY` the length is determined by `componentCount`; otherwise the length is 1. `normalized` property has no effect on array values: they always correspond to the integer values.",
+      "items": {
+        "type": "number"
+      },
+      "minItems": 1
+    },
+    "min": {
+      "type": "array",
+      "description": "Minimum allowed values for property values. Only applicable for fixed-size numeric element types (integer, floats and `ARRAY`s thereof). Both `min` and `max` arrays have the same length. When `type` is `ARRAY` the length is determined by `componentCount`; otherwise the length is 1. `normalized` property has no effect on array values: they always correspond to the integer values.",
+      "items": {
+        "type": "number"
+      },
+      "minItems": 1
+    },
+    "default": {
+      "type": ["boolean", "number", "string", "array"],
+      "description": "A default value to use when no value for this property is specified.  This happens only when using JSON encoding and `null` is used to indicate no value. If used, `optional` must be set to `true`. The data type here must match the property declaration. However, `BLOB` properties must be encoded in Base64 encoding since JSON does not support arbitrary binary sequences."
+    },
+    "optional": {
+      "type": "boolean",
+      "description": "If true, this property is optional and can take a null value. This is accomplished by not using a `bufferView` or by using a validity buffer. A validity bit set to 0 indicates the absence of data for that instance. In both cases, `optional` must be set to `true`.",
```

Comment:
Oh yeah this description is going to change a little bit today. See https://github.com/CesiumGS/3d-tiles/pull/434#discussion_r517724163.

I'll make this change soon in my branch and bump this comment when done.

Reply:
@sanjeetsuhag I pushed the changes, you can update this file now.